server/test/api/conventions.js

```diff
@@ -0,0 +1,122 @@
+/* jshint expr: true */
+/* global describe, it, beforeEach */
+
+var chai = require('chai');
+var expect = chai.expect;
+
+/** import test helpers */
+var helpers = require('./helpers');
+var uuid    = require('node-uuid');
+helpers.configure(chai);
+
+
+/**
+* The /cash/conventions API endpoint
+*
+* @desc This test suit is about the conventions payment in the primary cash
+*
+*/
+describe('The /cash/conventions API endpoint :: ', function () {
+  'use strict';
+
+  var agent = chai.request.agent(helpers.baseUrl);
+
+  /** login before each request */
+  beforeEach(helpers.login(agent));
+
+  /** Test with dates */
+  var date = new Date();
+  var futurDate = new Date(date.getYear() + 1900, date.getMonth(), date.getDay() + 10);
+
+  /** Primary Cash Object */
+  var primaryCash = {
+      uuid        : uuid(),
```

Comment:
This usage of `node-uuid` is deprecated.  [See this notice]https://www.npmjs.com/package/node-uuid#deprecated-apis).  The correct usage of `node-uuid` is like this:
```js
var uuid = require('node-uuid');
var primaryCash = {
  uuid : uuid.v4(),
  /* ... other properties ... */
};
```